src/moveit_commander/planning_scene_interface.py

```diff
@@ -105,6 +105,14 @@ def __make_mesh(self, name, pose, filename, scale = (1, 1, 1)):
         pyassimp.release(scene)
         return co
     
+    def __make_existing(self, name):
+        """
+        Create an empty Collision Object, used when the object already exists 
+        """
+        co = CollisionObject()
+        co.id = name
```

Comment:
I've just tried to do this by subscribing to /move_group/monitored_planning_scene and checking if such object is found in world.collision_objects before attaching it. However a simple subscriber is not enough since move_group only publishes a diff. Is there something lighter than merging each time the diffs with the known planning scene? Keep a list of known objects in attribute?